asserts/device_asserts.go

```diff
@@ -194,3 +194,61 @@ func assembleSerial(assert assertionBase) (Assertion, error) {
 		pubKey:        pubKey,
 	}, nil
 }
+
+// SerialRequest holds a serial-request assertion, which is a self-signed request to obtain a full device identity bound to the device public key.
+type SerialRequest struct {
+	assertionBase
+	pubKey PublicKey
+}
+
+// BrandID returns the brand identifier of the device making the request.
+func (sreq *SerialRequest) BrandID() string {
+	return sreq.HeaderString("brand-id")
+}
+
+// Model returns the model name identifier of the device making the request.
+func (sreq *SerialRequest) Model() string {
+	return sreq.HeaderString("model")
+}
+
+// NonceTicket returns the nonce/ticket for the request.
+func (sreq *SerialRequest) NonceTicket() string {
+	return sreq.HeaderString("nonce-ticket")
```

Comment:
This is a unique and non-guessable identifier for the request, which we can come back to the server and talk about N times. As such, it's not really a nonce as that term is used precisely to constrain the data to be used only once. How about "request-key"?